CMakeLists.txt

```diff
@@ -25,7 +25,7 @@ list(APPEND CMAKE_MODULE_PATH "${CMAKE_CURRENT_LIST_DIR}/cmake")
 # Set c++11 or higher
 include(EnableC++XX)
 # Set c11
-set(CMAKE_C_STANDARD 11)
```

Comment:
How that sneak in?

Reply:
We should probably check other projects for C++11 -v- 17

Reply:
The livox plugin requires c++17 to compile. Took a look at our implementation and found that c++17 doesn't break anything so it was safe to switch over

Reply:
To be fair i didn't test c++11 with livox, I decided to trust their Cmakelists

Reply:
Agreed, gazebo and PX4 are expected to have lurking c++11 binds due to the age of both projects. 

Also, i actually remember testing this entire thing with c++11 and the plugin complained IIRC, testing again now though to be sure

Reply:
I stand corrected, c++11 seems to work fine. Shall i switch over? @stevenmorphet-at

Reply:
For now, no. I'd like others to weigh in on what should be done that way.